### PR TITLE
Require Symfony 7.2

### PR DESCRIPTION
**Pull request by alamirault**

### CODE REVIEW

File: rector.php [hunk 1]
```diff
@@ -46,7 +46,7 @@
         SetList::PHP_83,
         PHPUnitSetList::PHPUNIT_90,
         PHPUnitSetList::PHPUNIT_CODE_QUALITY,
-        SymfonySetList::SYMFONY_71,
+        SymfonySetList::SYMFONY_72,
```
Comment (alamirault) [inline rector.php:49]: Need https://github.com/rectorphp/rector-symfony/pull/684

File: composer.json [hunk 2]
```diff
@@ -9,14 +9,14 @@
         "ext-mbstring": "*",
         "composer/semver": "^3.4.3",
         "ondram/ci-detector": "^4.2.0",
-        "symfony/config": "^7.1.1",
-        "symfony/console": "^7.1.5",
-        "symfony/dependency-injection": "^7.1.5",
-        "symfony/finder": "^7.1.4",
-        "symfony/options-resolver": "^7.1.1",
+        "symfony/config": "^7.2.0-RC1",
```
Comment (alamirault) [inline composer.json:12]: Replace `^7.2.0-RC1` to `^7.2.0` when availlable

**Comment by OskarStark** [comment]
Released!

https://github.com/symfony/symfony/releases/tag/v7.2.0

File: rector.php (hunk 1 above)
Comment (OskarStark) [reply rector.php:49]: Lets do this in a follow up PR?

**Comment by alamirault** [comment]
PR Updated

**Comment by alamirault** [comment]
Already done by https://github.com/OskarStark/doctor-rst/pull/1903

Bots are better than humans :see_no_evil:

File: rector.php (hunk 1 above)
Comment (OskarStark) [reply rector.php]: This is still open, right?

Comment (alamirault) [reply rector.php]: Rector 2.0 has been released, but it need to update phpstan 2.x Aldo.

And there is a conflict

Comment (xabbuh) [reply rector.php]: see #1928

**Pull request closed without merging**